### DISCUSSION TITLE
virus detected

### DISCUSSION THREAD

**Discussion by oneklema** [MustardChef/WSABuilds#243] [General]
Over the last few days I have been getting virus alerts from Windows security (Windows 11) virus scanner of a trojan horse and also a notification that the gms app is being blocked for security reasons. I have the most recent build installed and only a few apps that I use frequently on other android devices. The file is:
\AppData\Local\Packages\MicrosoftCorporationII.WindowsSubsystemForAndroid_8wekyb3d8bbwe\LocalCache\SharedTemp\quarantine\quaEE68.tmp
and the trojan detected is Trojan:Win32/Cloxer

Any thoughts if this is a real threat?

**Comment by abdullahmeric** [comment]
I got the same warning for the following file AppData\Local\Packages\MicrosoftCorporationII.WindowsSubsystemForAndroid_8wekyb3d8bbwe\LocalCache\SharedTemp\quarantine\qua26C6.tmp

**Comment by MustardChef** [comment]
This should not be remotely possible. Everything was built without any modifications.
This is most definitely a false positive.
There is not enough evidence to conclude that the WSABuilds builds are malware/virus.
Rest assured that there is no threat to your PC.

**Comment by MustardChef** [comment]
If there are more legitimate causes for concern, I am here to listen.

This repo is switching to GitHub Actions workflow, for increased transparency and security. It should also ease some concerns regarding malware.
